lasagne/tests/layers/test_pool.py

```diff
@@ -339,6 +339,56 @@ def test_get_output_shape_for(self, input_shape):
         except NotImplementedError:
             pytest.skip()
 
+    def test_not_implemented(self):
+        try:
+            from lasagne.layers.cuda_convnet import MaxPool2DCCLayer
+        except ImportError:
+            pytest.skip("cuda_convnet not available")
+
+        input_layer = self.input_layer((128, 4, 12, 12))
+
+        with pytest.raises(RuntimeError) as exc:
+            layer = MaxPool2DCCLayer(input_layer, pool_size=2, pad=2)
+        assert "MaxPool2DCCLayer does not support padding" in exc.value.args[0]
```

Comment:
Hmm, funny. The original code has a `start` parameter that can be used for padding or cropping: https://code.google.com/p/cuda-convnet/wiki/LayerParams#Local_pooling_layer
But that's far outside this PR.